tests/test_cli.py

```diff
@@ -0,0 +1,4 @@
+def test_argparser():
+    sys.argv = "covert enc --recipient asdf -r asdf -Arrp recipient1".split()
```

Comment:
Should the `-Arrp` part be expecting two additional recipients? This test should lead to the missing parameter error. I suggest splitting to two tests, where one adds ` recipient2` at the end, and the other keeps this exact same string.

I believe you can do the checking for error by
```python
with pytest.raises(SystemExit):  # sys.exit() raises SystemExit exception which is received by pytest here
  a = argparser()
```
... and perhaps also checking that the error message gets printed on stderr (see pytest docs or google for more information).